account_invoice_triple_discount/models/account_move.py

```diff
@@ -16,29 +16,28 @@ def _recompute_tax_lines(self, **kwargs):
         restored after the original process is done
         """
         old_values_by_line_id = {}
-        digits = self.invoice_line_ids._fields["price_unit"]._digits
-        self.invoice_line_ids._fields["price_unit"]._digits = (16, 16)
-        for line in self.invoice_line_ids:
+        lines = self.invoice_line_ids or self.line_ids
```

Comment:
I think it would be better to check if it `is_invoice` instead.

Reply:
Working on the broken test, is_invoice returns True but without invoice_line_ids. So in this case it will not fix the test